issues_test.go

```diff
@@ -112,3 +120,57 @@ func TestIssue43(t *testing.T) {
 		}
 	})
 }
+
+func TestIssue81(t *testing.T) {
```

Comment:
The meaning of `TestIssueXXX` is used to indicate that the bug has been fixed, in line with the official Go repository. (In fact, it is created for each file, but because of the volume, it is created as a single function.  e.g. [issue21104_test.go](https://github.com/golang/go/blob/master/src/crypto/issue21104_test.go))

I will correct this on our side for clarity later.